charts/kusion/README.md

```diff
@@ -63,16 +69,16 @@ helm install kusion-release ./kusion
 First, you need to base64 encode your kubeconfig files.
 
 ```shell
-KUBECONFIG_CONTENT1=$(base64 -w 0 /path/to/your/kubeconfig1)
-KUBECONFIG_CONTENT2=$(base64 -w 0 /path/to/your/kubeconfig2)
+KUBECONFIG_CONTENT1=$(cat /path/to/your/kubeconfig-1 | base64 | tr -d '\n')
+KUBECONFIG_CONTENT2=$(cat /path/to/your/kubeconfig-2 | base64 | tr -d '\n')
 ```
 
 Then you can use the following command to install the latest version of Kusion.
 
 ```shell
 helm install kusion-release kusionstack/kusion \
---set kubeconfig.kubeConfigs.kubeconfig0="$KUBECONFIG_CONTENT1" \
```

Comment:
The kubeconfig key naming convention change from 'kubeconfig0' to 'kubeconfig-0' is good for consistency, but should be clearly documented as a breaking change for users upgrading from previous versions